code/client/include/Client.h

```diff
@@ -0,0 +1,29 @@
+#ifndef COLLATZ_CONJECTURE_CLIENT_H
+#define COLLATZ_CONJECTURE_CLIENT_H
+
+#include <yarp/os/all.h>
+#include <vocabs.hpp>
+#include <ClientIDLInterface.h>
+class Client : public yarp::os::RFModule, ClientIDLInterface{
+private:
+
+    yarp::os::Port rpcPort;
```

Comment:
This port is then attached as server via the yarp().attachAsServer(Port) method. I have followed the tutorial for the implementation of Thrift interface in this way The rpcPort here is the one used to send commands from command line. Is there a better way to do this?